src/modules/functions.js

```diff
@@ -52,10 +52,33 @@ const updateIds = () => {
   return tasks;
 };
 
+const checkBox = (e) => {
+  tasks = getLocalStorage();
+  const task = e.target;
+  const index = Number(task.id.split('_')[1]);
+
+  if (tasks[index - 1].completed === false) {
+    tasks[index - 1].completed = true;
+  } else if (tasks[index - 1].completed === true) {
+    tasks[index - 1].completed = false;
+  }
```

Comment:
- [ ] Please the task object's values for completed are updated upon user actions but for you when I check the box the completed value remains `false`  it's supposed to change to `true` when the box is checking :upside_down_face: :upside_down_face:, please kindly fix that :grin: :grin:

Reply:
Hi @cynthiainga, thank you for the feedback, I've recorded a loom video showing the function, hopefully, it clears this misunderstanding, thank you very much 😃 
Here's the [loom video](https://www.loom.com/share/274ff09a4d8c4cd4966d79ee154d6d14).

Reply:
Hi @lucas-crodrigues, I understand what's wrong. But I think it should be better when you refresh the browser the localStorage update automatically. You can create a function to update the storage :grin: :grin: